### ISSUE TITLE
Broken since 1.3.0 with mongodb@1.x

### ISSUE THREAD

**Issue by jdesboeufs** [kcbanner/connect-mongo#228]
https://github.com/kcbanner/connect-mongo/issues/179#issuecomment-235287026

Current implementation use `collection.update()` response, which differ from `mongodb@1.x`.

**Comment by jdesboeufs** [comment]
Fixed in upcoming `1.3.2`.

**Comment by natiz** [comment]
I am still getting this issue in 1.3.2?

```
TypeError: Cannot read property 'upserted' of undefined
    at .../node_modules/connect-mongo/src/index.js:252:36
```
The content of responseArray seems to be: ```[ { ok: 1, nModified: 1, n: 1 } ]```

```
    "mongojs": "~2.4.0",
    "connect-mongo": "~1.3.2",
```

```
brew info mongodb
mongodb: stable 3.2.0 (bottled)
```
